features/filter_recipes.feature

```diff
@@ -7,52 +7,46 @@ Feature: Display filtered recipes to user sorted into Breakfast, Lunch & Dinner
 Background: recipes are retrieved by the API from a database 
 
   Given the following recipes exist:
-  |title			|day|slot|calories	|readyminutes|price	|
-  |Eggs Benedict	|1	|1	 |200		|45			 |2		|
-  |Sweet & Sour Soup|1	|2	 |340		|50			 |3		|
-  |Mushroom Risotto	|1	|3	 |200		|75			 |8		|
-  |Waffles			|2	|1	 |200		|45			 |8		|
-  |Salad			|2	|2	 |200		|45			 |8		|
-  |Burrito			|2	|3	 |200		|45			 |8		|
-  |Grapes			|3	|1	 |200		|45			 |8		|
-  |Cherries			|3	|2	 |200		|45			 |8		|
-  |Milk				|3	|3	 |200		|45			 |8		|
-
-  And I am on the GrocerPlus home page
+  |daily|slot|id	|title				|calories	|readyInMinutes	|price	|
+  |1	|1	 |123456|Eggs Benedict		|200		|45			 	|2		|
+  |1	|2	 |123456|Sweet & Sour Soup	|340		|50			 	|3		|
+  |1	|3	 |123456|Mushroom Risotto	|200		|75			 	|8		|
+  |2	|1	 |123456|Waffles			|200		|45			 	|8		|
+  |2	|2	 |123456|Salad				|200		|45			 	|8		|
+  |2	|3	 |123456|Burrito			|200		|45			 	|8		|
+  |3	|1	 |123456|Grapes				|200		|45			 	|8		|
+  |3	|2	 |123456|Cherries			|200		|45			 	|8		|
+  |3	|3	 |123456|Milk				|200		|45			 	|8		|
+
+  And I am on the home page
   Then 9 seed recipes should exist 
 
 Scenario: API call returns 3 breakfast recipes
-  When I view the meals page
-    And view the Breakfast section
-  Then I should see "Eggs Benedict"
-    And I should see "Waffles"
-    And I should see "Grapes"
+  When API call returns 3 breakfast recipes
+  Then I should see "Eggs Benedict" in breakfast
+    And I should see "Waffles" in breakfast
+    And I should see "Grapes" in breakfast
 
 Scenario: API call returns 3 lunch recipes
-  When I view the meals page
-    And view the Lunch section
-  Then I should see "Sweet & Sour Soup"
-    And I should see "Salad"
-    And I should see "Cherries"
+  When API call returns 3 lunch recipes
+  Then I should see "Sweet & Sour Soup" in lunch
+    And I should see "Salad" in lunch
+    And I should see "Cherries" in lunch
 
 Scenario: API call returns 3 dinner recipes
-  When I view the meals page
-    And view the Dinner section
-  Then I should see "Mushroom Risotto"
-    And I should see "Burrito"
-    And I should see "Milk"
+  When API call returns 3 dinner recipes
+  Then I should see "Mushroom Risotto" in dinner
+    And I should see "Burrito" in dinner
+    And I should see "Milk" in dinner
 
 Scenario: API call returns no breakfast recipes
-  When I view the meals page
-    And view the Breakfast section
-  Then I should see an empty section
+  When API call returns no breakfast recipes
+  Then I should see nothing in breakfast
```

Comment:
In this Scenario and the other two (returns no lunch recipes, returns no dinner recipes), maybe include the steps for "should see lunch" and "should see dinner"

Reply:
resolved